sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOTest.java

```diff
@@ -976,7 +977,7 @@ public void testBuildSourceDisplayData() {
 
     BigQueryIO.Read.Bound read = BigQueryIO.Read
         .from(tableSpec)
-        .fromQuery("myQuery")
+        .fromQuery(StaticValueProvider.of("myQuery"))
```

Comment:
I'm optimistic this change (and similar above/below) is unnecessary, because BigQuery.Read has the swap-String-for-SVP already?

(BigQueryTableSource need not have any swaps as it's an internal API.)